stream_batch.go

```diff
@@ -299,7 +299,7 @@ func (sbr *streamBatchReader) reachedEndOfTheGeneration(windowEnd gocql.UUID) bo
 
 func (sbr *streamBatchReader) close(processUntil gocql.UUID) {
 	sbr.endTimestamp.Store(processUntil)
-	sbr.interruptCh <- struct{}{}
```

Comment:
Now, if you call `sbr.close()` with timestamp from future, it will spin in the following cycle without break consuming all cpu available:
https://github.com/scylladb/scylla-cdc-go/blob/c4f1a40bf658e5b9651964c6b8d9e13406ac7383/stream_batch.go#L153-L164